trunk/SUAVE/Methods/Weights/Buildups/Electric_Lift_Cruise/empty.py

```diff
@@ -0,0 +1,163 @@
+            
+        Calculates the empty fuselage mass for an electric stopped rotor including
+        seats, avionics, servomotors, ballistic recovery system, rotor and hub
+        assembly, transmission, and landing gear. Additionally incorporates
+        results of the following common buildup scripts:
+            fuselage,py
+            prop.py
+            wing.py
+            wiring.py
+        Originally written as part of an AA 290 project inteded for trade study
+        of the Electric Stopped Rotor along with the following defined SUAVE config types:
+            Electric Multicopter
+            Electric Vectored_Thrust
+            
+        Sources:
+        Project Vahana Conceptual Trade Study
+        https://github.com/VahanaOpenSource
+        
+        Inputs:
+            config                          SUAVE Config Data Structure
+            speed_of_sound                  Local Speed of Sound                [m/s]
+            maximumTipMach                  Allowable Tip Mach Number           [Unitless]
+            disk_area_factor                Inverse of Disk Area Efficiency     [Unitless]
+            max_thrust_to_weight_ratio      Allowable Thrust to Weight Ratio    [Unitless]
+            motor_efficiency                Motor Efficiency                    [Unitless]
+        
+        Outputs:
+            output:                         Data Dictionary of Component Masses       [kg]
+    """
+
+    output = Data()
+
+    #-------------------------------------------------------------------------------
+    # Unpack Inputs
+    #-------------------------------------------------------------------------------
+    rRotor              = config.propulsors.propulsor.rotor.tip_radius 
+    rotor_bladeSol      = config.propulsors.propulsor.rotor.blade_solidity    
+    rPropThrust         = config.propulsors.propulsor.propeller.tip_radius
+    mBattery            = config.propulsors.propulsor.battery.mass_properties.mass
+    mPayload            = config.propulsors.propulsor.payload.mass_properties.mass
+    MTOW                = config.mass_properties.max_takeoff
+    nLiftProps          = config.propulsors.propulsor.number_of_engines_lift
+    nThrustProps        = config.propulsors.propulsor.number_of_engines_forward
+    nLiftBlades         = config.propulsors.propulsor.rotor.number_blades
+    nThrustBlades       = config.propulsors.propulsor.propeller.number_blades
+    fLength             = config.fuselages.fuselage.lengths.total
+    fWidth              = config.fuselages.fuselage.width
+    fHeight             = config.fuselages.fuselage.heights.maximum
+    maxSpan             = config.wings['main_wing'].spans.projected
+     
+    tipMach             = max_tip_mach
+    k                   = disk_area_factor
+    ToverW              = max_thrust_to_weight_ratio
+    etaMotor            = motor_efficiency
+
+    output.payload      = mPayload * Units.kg
+    output.seats        = 30. *Units.kg
```

Comment:
Lots of hardcoded values here. Is this appropriate? Especially for seats and other things that I would expect to vary with passengers.

Reply:
@JTrentSmart indicated that he has a branch with concise and consolidated scripts (feature-eVTOL_weights" . I plan to do this after this PR is done